examples/gallery/lines/quoted_lines.py

```diff
@@ -0,0 +1,95 @@
+"""
+Quoted lines
+------------
+To plot a so-called *quoted line*, i.e., labels along a line
+or curve, use the ``style`` parameter of the
+:meth:`pygmt.Figure.plot` method with the argument ``"q"`` and the
+desired modifiers. A colon (``":"``) is used to separate the
+algorithm settings from the label information.
+This example shows how to adjust the labels.
+For modifying the base line via the ``pen`` parameter, see the
```

Comment:
```suggestion
For modifying the baseline via the ``pen`` parameter, see the
```

Reply:
All occurencies are updated in commit https://github.com/GenericMappingTools/pygmt/pull/2563/commits/c3b04926e0eaf01d01edee45bde5bda56711cb17.